hyperspy/_components/pes_voigt.py

```diff
@@ -0,0 +1,331 @@
+# -*- coding: utf-8 -*-
+# Copyright 2007-2016 The HyperSpy developers
+#
+# This file is part of  HyperSpy.
+#
+#  HyperSpy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+#  HyperSpy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with  HyperSpy.  If not, see <http://www.gnu.org/licenses/>.
+
+import numpy as np
+import math
+
+from hyperspy.component import Component
+from hyperspy._components.gaussian import _estimate_gaussian_parameters
+
+
+sqrt2pi = math.sqrt(2 * math.pi)
+sigma2fwhm = 2 * math.sqrt(2 * math.log(2))
+
+
+def voigt(x, FWHM=1, gamma=1, center=0, scale=1):
+    r"""Voigt lineshape.
+
```

Comment:
stray 'r' here

Reply:
This `r` is necessary to render the latex properly.

Reply:
Ah - my bad, reviewing = learning = :)